src/ducks/settings/CategoryAlerts/CategoryAlertEditModal.jsx

```diff
@@ -146,86 +222,34 @@ const fieldSpecs = {
  */
 const CategoryAlertEditModal = translate()(
   ({ initialAlert, onEditAlert, onDismiss, t }) => {
-    const [alert, setAlert] = useState(initialAlert)
-    const [choosing, setChoosing] = useState(null)
-
-    const handleChoosingCancel = () => {
-      setChoosing(null)
-    }
-
-    const handleChangeField = (name, val) => {
-      const updater = fieldSpecs[name].updater
-      const updatedAlert = updater(alert, val)
-      setAlert(updatedAlert)
-    }
-
-    const handleRequestChooseField = name => {
-      const fieldSpec = fieldSpecs[name]
-      setChoosing({
-        type: fieldSpec.type,
-        value: fieldSpec.getValue(alert),
-        onSelect: val => {
-          setChoosing(null)
-          handleChangeField(name, val)
-        },
-        onCancel: handleChoosingCancel
-      })
-    }
-
-    const handleConfirmEdit = () => {
-      onEditAlert(alert)
-    }
-
     const modalTitle = t('Settings.budget-category-alerts.edit.modal-title')
+    const okButtonLabel = (props, doc) =>
+      doc.id !== undefined
+        ? t('Settings.budget-category-alerts.edit.update-ok')
+        : t('Settings.budget-category-alerts.edit.create-ok')
```

Comment:
`props` doesn't seem to be used in this function

Reply:
This was the API of EditionModal, I changed it to only use doc. We will add props later if needeed.